src/dataset-builder/DatasetBuilder.ts

```diff
@@ -571,8 +572,10 @@ export const DatasetBuilderContents = ({
 
   useEffect(() => {
     requestValid &&
-      setDatasetRequestParticipantCount(async () =>
-        DataRepo().dataset(dataset.id).getSnapshotBuilderCount(createSnapshotBuilderCountRequest(allCohorts))
+      setDatasetRequestParticipantCount(
+        withErrorReporting('Error generating dataset counts')(async () =>
```

Comment:
Thoughts on changing the message to 
`Error generating participant count`

Reply:
Sure! Done.